src/Microsoft.VisualStudio.Validation.Tests/AssumesTests.cs

```diff
@@ -1,4 +1,6 @@
-﻿using System;
+﻿#nullable enable
```

Comment:
Consider enabling this at the project level with `<Nullable>true</Nullable>` then removing all these pragmas. That will ensure code in new files are analyzed by default.

Reply:
I tried that before sending this PR. But it didn't appear to work. Here you can see from my screenshot that I have added the property to the project file, and removed the `#nullable enable` from the soruce file, and yet I have many errors and warnings because the compiler doesn't consider nullable to be turned on:

![image](https://user-images.githubusercontent.com/3548/63267718-e9123400-c24f-11e9-9a42-071545425160.png)

Reply:
Reloading the solution doesn't resolve the issue either. Neither does deleting the `.vs` folder. Are we perhaps trying the wrong project property name?

Reply:
Ah, the problem was the property *value*. `true` isn't an accepted value. It  must be 'disable', 'enable', 'warnings' or 'annotations'.